.github/workflows/CI.yml

```diff
@@ -100,9 +100,12 @@ jobs:
         sudo apt install graphviz
         python -m pip install --upgrade pip
         pip install --user -r requirements.txt
+        mkdir $HOME/bin
+        echo -e '#!/bin/sh\ndocker run -v $HOME:$HOME -w $PWD componolit/ci:gnat-community-2019 gprbuild $@' > $HOME/bin/gprbuild
+        chmod +x $HOME/bin/gprbuild
```

Comment:
Is there any reason not to put this into `~/.local/bin`, which is already existing?

Reply:
`~/.local/bin´ is cached based on the used python version and restored before each CI run. The current assumption is that this directory only contains executables created by pip.

In the current version of the CI config it would make no difference which directory is used. But I think it could lead to errors, if the cache is used in the future and there are unexpected executables in it.

Reply:
Ah okay. I wasn't aware of that caching. In this case it makes sense to use a separate directory.